bin/pp_ast.ml

```diff
@@ -97,6 +97,12 @@ let loc_mode =
   in
   named (fun x -> `Loc_mode x) Cmdliner.Arg.(value & vflag `Short [ full_locs ])
 
+let full_consts =
```

Comment:
I don't think this is necessary, since the only other field is the location, we can make the `show-loc` flag control whether we skip the constant record or not, similar to what we do with over records with a `_desc` field.